src/signals/incident-management/containers/Dashboard/components/Filter/Filter.tsx

```diff
@@ -0,0 +1,79 @@
+// SPDX-License-Identifier: MPL-2.0
+// Copyright (C) 2023 Gemeente Amsterdam
+import { useCallback, useEffect, useRef, useState } from 'react'
+
+import { useClickOutside } from '@amsterdam/asc-ui'
+import { useForm, FormProvider } from 'react-hook-form'
+import { generateParams } from 'shared/services/api/api'
+
+import SelectList from './SelectList'
+import { FilterContainer } from './styled'
+import type { Option } from './types'
+
+type Props = {
+  callback?: (queryString: string) => void
+}
+
+export const Filter = ({ callback = () => {} }: Props) => {
```

Comment:
Why is the callback optional and has a default value? Should we just conditionally call it or make it required?

Reply:
make it required I could say when dashboard api is implemented